utils.py

```diff
@@ -69,8 +53,16 @@ def annotate_wheels(packages):
 
 def get_top_packages():
     print('Getting packages...')
-    packages = req_rpc('top_packages')
-    return [{'name': n, 'downloads': d} for n, d in packages]
+
+    with open('top-pypi-packages.json') as data_file:
```

Comment:
Currently the download of this file only occurs on Travis, so this will fail on deploy (which looking at the repo appears to be performed via the `Makefile`).

Since the site is statically generated, it seems fine to just fetch the file directly from GitHub in this method, and then drop the change to `.travis.yml`.

Reply:
Thanks for that -- yes, `Makefile` would be the ideal place to fetch the file.

Reply:
Updated!